schema/verifiable-presentation/verifiable-presentation-schema.json

```diff
@@ -56,7 +53,8 @@
           ]
         },
         "created": {
-          "type": "string"
+          "type": "string",
+          "pattern": ".*(Z|(\\+|-)[0-9][0-9]:[0-9][0-9])"
```

Comment:
Just marking with the same comment as above

Reply:
Same issue, use the regex here instead:

https://www.w3.org/TR/vc-data-model-2.0/#example-regular-expression-to-detect-a-valid-xml-schema-1-1-part-2-datetimestamp